### PR TITLE
Unity ads

### PR DESCRIPTION
**Pull request by nooralibutt**
implements unity ads
updates call back methods
updates examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,63 +87,108 @@ void main() async {
...
-EasyAds.instance.disposeInterstitialAd();
+## Banner Ads
 
-OR
+#### Declare Banner ad
+
+Declare banner ad instance variable and banner ad placement ids
+
+```dart
+class _CountryDetailScreenState extends State<CountryDetailScreen> {
```
Comment (ahmadkhandev) [inline README.md:145]: it is okay but i would rather use _ExampleScreenState

File: README.md [hunk 2]
```diff
@@ -86,63 +87,108 @@ void main() async {
...
+```
 
-EasyAds.instance.disposeInterstitialAd();
+## Banner Ads
 
-OR
+#### Declare Banner ad
+
+Declare banner ad instance variable and banner ad placement ids
+
```
Comment (ahmadkhandev) [inline README.md:143]: should I add the Admob banner example?

File: example/lib/main.dart [hunk 3]
```diff
@@ -103,17 +122,43 @@ class _CountryListScreenState extends State<CountryListScreen> {
...
 
+  @override
+  State<CountryDetailScreen> createState() => _CountryDetailScreenState();
+}
+
+class _CountryDetailScreenState extends State<CountryDetailScreen> {
+  static final _unityBannerPlacementId =
+      Platform.isAndroid ? 'Banner_Android' : 'Banner_iOS';
+
+  late final EasyUnityBannerAd _bannerAd;
```
Comment (ahmadkhandev) [inline example/lib/main.dart:139]: so till now, we haven't added the functionality to use banner ads from one file?

File: lib/src/easy_admob/easy_admob_banner_ad.dart [hunk 4]
```diff
@@ -64,14 +63,14 @@ class EasyAdmobBannerAd extends EasyAdBase {
...
       return const SizedBox();
     }
 
-    return SizedBox(
+    return Container(
+      alignment: Alignment.center,
       child: AdWidget(ad: ad),
-      height: 50,
-      width: double.infinity,
+      height: adSize.height.toDouble(),
```
Comment (ahmadkhandev) [inline lib/src/easy_admob/easy_admob_banner_ad.dart:72]: so it gives the default lengths of AdSize.banner?

**Review by ahmadkhandev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nooralibutt) [reply README.md:145]: Yes, better. When you update read me, you consider that

File: README.md (hunk 2 above)
Comment (nooralibutt) [reply README.md:143]: yes please

File: example/lib/main.dart (hunk 3 above)
Comment (nooralibutt) [reply example/lib/main.dart:139]: no, not yet, reason cz I will integrate all the potential ad services and then I'd check if we actually can write single banner widget

File: lib/src/easy_admob/easy_admob_banner_ad.dart (hunk 4 above)
Comment (nooralibutt) [reply lib/src/easy_admob/easy_admob_banner_ad.dart:72]: yes whatever user selected

**Pull request merged**